app/models/user.py

```diff
@@ -451,6 +453,11 @@ def rocket_chat_username(self):
     def rocket_chat_password(self):
         return get_serializer().dumps(f'rocket_chat_user_{self.id}')
 
+    @property
+    def has_group_role(self):
+        group = UsersGroupsRoles.query.filter_by(user_id=self.id)
```

Comment:
This shouldn't be added in user. Secondly this gives every user in group team the access level of organizer